### DISCUSSION TITLE
Issue: Reaction tooltip can be hidden under sidebar during code review

### DISCUSSION THREAD

**Discussion by curusarn** [community/community#28303] [Accessibility] [answered]
When doing a code review, the reactions tooltip is hidden under the sidebar when "reaction icon" under message is used:
![Screenshot from 2022-08-04 11-19-35](https://user-images.githubusercontent.com/10132717/182814506-5465ddb9-3246-497e-8cc2-dba8d8cdfd2c.png)

Using the dedicated "reaction button" works fine:
![Screenshot from 2022-08-04 11-19-17](https://user-images.githubusercontent.com/10132717/182814541-d2267126-8884-4423-8f33-2793c7d26aed.png)

**Comment by jfullstackdev** [comment, marked as answer]
yup, this behavior is already reported

**Comment by cehfisher** [comment]
Hi @curusarn - I wanted to follow up on this stale issue. Have the recent theme changes resolved this issue for you? If so, could you please close the issue? If not, what additional changes are needed? (cc @queenofcorgis)
